Turn on functionalization by default in functorch

eellison opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #84435

I talked to @SherlockNoMad abt this PR and we agreed prior to brian coming back it was worth disabling this test for getting functionalization on (and that is already the state of torchdynamo)

eellison committed:
[Testing][Not For land] turn on functionalization

[ghstack-poisoned]

eellison committed:
Update on "[Testing][Not For land] turn on functionalization"




[ghstack-poisoned]

Chillee reviewed (approved):
So this is easy, and makes sense :)

Oh, I guess this is already turned on by default inside Inductor, so yeah, makes sense to do it here.

eellison committed:
Update on "Turn on functionalization by default in functorch"


I talked to SherlockNoMad abt this PR and we agreed prior to brian coming back it was worth disabling this test for getting functionalization on (and that is already the state of torchdynamo)


[ghstack-poisoned]

eellison commented:
@pytorchbot merge --green

eellison commented:
@pytorchbot merge
